content/blog/aneesh.md

```diff
@@ -0,0 +1,37 @@
+---
+title: My experience with FOSS Overflow
+author: Aneesh Tickoo
+date: '2022-01-21'
+---
+
+# Introduction
+
+To summarize my project in a nutshell, upload group photographs, categorize them by faces using ML, and easily share them. 🤳🤖
+
+This year in [Foss Overflow 2021](https://fossoverflow.dev/), I worked on [Photo Sharing App](https://github.com/OpenLake/Photo-Sharing-App) project. It is a web app that allows a user to get group photos that include them without having to keep or sort through all of the photos, saving them time and storage space. When a photo is uploaded, the app identifies the faces in it and arranges them according to the users who have been identified. Each identified user gets a separate album with all their photos allowing them to get the photos they need seamlessly.
+
+## How was my experience with FOSSOverflow'21?
+
+My project got off to a rocky start, and it wasn't until I discussed my concerns with my mentor that I discovered an efficient and improved approach to developing software. My mentor's crucial advice is still fresh in my mind. *Planning is critical before you write code. There's no point in running in the wrong direction.* My mentor exposed me to many new topics, and it was fun to explore project completion ideas together. I'm thrilled I was able to contribute to a solution to a real-world problem. Thank you for assisting me through the development process and beyond, [Kumar Shivendu](https://github.com/KShivendu).✌😎
+
+## Why did I select Photo Sharing App as my project for FOSSOverflow'21?
+
+I am constantly exploring Machine Learning topics, and this project was right up my alley. I have some past experience with image processing (basic MNIST dataset) and wanted to develop my skills. I enjoyed using and learning new abilities on a real-world assignment. Overall, this project has given me a greater understanding of the development and the realm of OpenSource. I'd like to continue working on this project even after the event.💻✨
+
+## What did I like about FOSSOverflow'21?
+
+The fact that FOSSOverflow'21 is an actual simulation of GSoC gives valuable insight into the challenges that may arise during the actual event. The challenges begin by writing your proposal, then adhering to your plan, and finally finishing your responsibilities on time. Apart from that, my interactions with my mentor exposed me to new and better ideas. Overall, it was a fantastic experience. Thank you, organizers and [Kumar Shivendu](https://github.com/KShivendu) for this wonderful experience.🎉✨
+
+
+## Achievements during FOSSOverflow'21
+Some of my achievements during the project:🦾
+
+1. Developing a basic UI for users
+2. Allowing users of a single room to upload their photos
+3. Implementing OpenCV's face recognition library to encode faces
+4. Using DBSCAN to cluster faces together
+5. Finally, showing face albums in order of their similarities to each other
+
+and much more...
```

Comment:
Add a few screenshots and some technical details about how the project works. What are the different steps that you took while grouping the photos? (jut an overview)